src/lib/storage/proposal/settings.rb

```diff
@@ -82,6 +84,7 @@ def initialize
           # Not yet in control.xml
           @home_min_size                 = DiskSize.GiB(10)
           @home_max_size                 = DiskSize.unlimited
+          @useful_free_space_min_size    = DiskSize.MiB(30)
```

Comment:
Adding the value to `Proposal::Settings` looked like the cheapest way to share it between different proposal sub-components. We don't need to put it in control.xml right now, but having it in `Settings` allows us to do it configurable in the future. I like that flexibility and I expect responsibility from PMs if we document which options are dangerous.

Would it be enough if I make more obvious (using constants, for example) the fact that we don't plan to add `settings.useful_free_space_min_size` to control.xml for the time being?

Of course, there are many other options. Another equally cheap alternative would be to extract `TINY_FREE_CHUNK` (most likely together with other constants like `VOLUME_GROUP_SYSTEM`) to `/lib/storage/proposal/constants.rb` to make easy to share it between proposal subcomponents.